doc/source/install.rst

```diff
@@ -72,6 +72,285 @@ and start hacking it with:
+      (description "Get @code{arXiv.org} metadata in @code{BibTeX} format")
+      (license license:bsd-3)))
+  
+  ;; added
+  (define python-habanero
+    (package
+      (name "python-habanero")
+      (version "1.2.3")
+      (source (origin
+                (method url-fetch)
+                (uri (pypi-uri "habanero" version))
+                (sha256
+                 (base32
+                  "02792xxr4mwa17khw6szmpvdlck28sz0npfw57c2rb9dnh0rwvqr"))))
+      (build-system pyproject-build-system)
+      (arguments '(#:tests? #f))
+      (propagated-inputs (list python-requests python-tqdm))
+      (native-inputs (list python-pytest))
+      (home-page "https://github.com/sckott/habanero")
+      (synopsis "Low Level Client for Crossref Search API")
+      (description "Low Level Client for Crossref Search API")
+      (license license:expat)))
+  
+  (package
+   (name "python-papis")
+   (version "0.13")
+   (source (origin
+            (method url-fetch)
+            (uri (pypi-uri "papis" version))
+            (sha256
+             (base32
+              "19v5r6761b0pm63xr1kxmlw7cn7cijn1dnbpbdybc6rn72lnlnpk"))))
+   (build-system pyproject-build-system)
+   (arguments '(#:tests? #f))
+   (propagated-inputs (list python-arxiv2bib
+                            python-beautifulsoup4
+                            python-bibtexparser ;added
+                            python-chardet
+                            python-click
+                            python-colorama
+                            python-dominate
+                            python-filetype
+                            python-habanero ;added
+                            python-isbnlib
+                            python-prompt-toolkit
+                            python-pygments
+                            python-pyparsing
+                            python-doi
+                            python-pyyaml
+                            python-requests
+                            python-slugify ;added
+                            python-stevedore
+                            python-tqdm
+                            python-typing-extensions))
+   (native-inputs (list python-flake8
+                        python-flake8-bugbear
+                        python-flake8-quotes
+                        python-mypy
+                        python-pep8-naming
+                        python-pylint
+                        python-pytest
+                        python-pytest-cov
+                        python-coveralls
+                        python-lsp-server
+                        python-sphinx-click
+                        python-sphinx-rtd-theme
+                        python-types-beautifulsoup4
+                        python-types-pygments
+                        python-types-python-slugify
+                        python-types-pyyaml
+                        python-types-requests
+                        python-types-tqdm))
```

Comment:
These type annotation packages are only required for testing. Do they need to be in the recipe? Who is this intended for: developers or users?

Reply:
The default workflow for guix's "pyproject-build-system" includes testing so ideally these packages would be used during the make process. I, however, could not build papis with passing all its tests so I temporary switched it off. Later, if time allows, I would actually make sure that all tests are passing as for most of the other packages that had to be defined prior to papis package because they are not (yet) in the main guix repo.

Reply:
Fair enough! Thank you for explaining.